test/gtest/ucs/test_mpool.cc

```diff
@@ -14,6 +14,16 @@ extern "C" {
 #include <queue>
 
 class test_mpool : public ucs::test {
+public:
+    typedef struct {
+        const size_t                elem_size;
+        const unsigned              num_elems_first_chunk;
+        const unsigned              total_num_elems;
+        const double                grow_factor;
+        const std::vector<unsigned> num_elems_per_chunk;
+        ucs_mpool_t                 mp;
+    } test_mpool_grow_factor_case_t;
```

Comment:
should extend test_mpool in new test class and put test_mpool_grow_factor_case_t and check_chunk_elems_num in it - without changing the base class test_mpool

Reply:
changed accordingly